assets/php/class-wgpb-block-library.php

```diff
@@ -354,6 +377,14 @@ public static function print_script_settings() {
 		$code           = get_woocommerce_currency();
 		$product_counts = wp_count_posts( 'product' );
 
+		$product_categories = get_terms(
+			'product_cat',
+			array(
+				'hide_empty' => false,
```

Comment:
Any reason behind adding in these two arguments here? Guessing get_terms can be filtered to over-ride, just more curious really.

Reply:
`hide_empty` so that we always get all terms, `pad_counts`… I don't remember. WC core used it so I copied it 😅